src/providers/ipa/ipa_access.c

```diff
@@ -470,83 +471,63 @@ static errno_t ipa_fetch_hbac_recv(struct tevent_req *req)
     return EOK;
 }
 
-static errno_t ipa_save_hbac(struct sss_domain_info *domain,
+static errno_t ipa_save_hbac(TALLOC_CTX *mem_ctx,
+                             struct sss_domain_info *domain,
                              struct ipa_fetch_hbac_state *state)
```

Comment:
Correct me if I'm mistaken, but it's not exactly a leak as it'll be freed when state (which is used as mem_ctx) is freed.

As the code end up being removed I personally wouldn't bother adding a temporary context there.

But you have the last sentence, so, please, let me know!